stripe/src/main/java/com/stripe/android/StripePaymentController.kt

```diff
@@ -546,6 +546,23 @@ internal class StripePaymentController internal constructor(
                         enableLogging = enableLogging
                     )
                 }
+                is StripeIntent.NextActionData.DisplayOxxoDetails -> {
+                    // TODO(smaskell): add analytics event
+                    if (nextActionData.hostedVoucherUrl != null) {
+                        beginWebAuth(
+                            host,
+                            getRequestCode(stripeIntent),
+                            stripeIntent.clientSecret.orEmpty(),
+                            nextActionData.hostedVoucherUrl,
+                            requestOptions.stripeAccount,
+                            enableLogging = enableLogging,
+                            cancelOnUserNavigation = false
+                        )
+                    } else {
+                        // TODO(smaskell): Determine how to handle missing URL
```

Comment:
@pololi-stripe any thoughts on how we should handle the case where the voucher url is not present? Right now it's just bypassing any confirmation

Reply:
Sorry, I didn't handle this part in iOS. As I discussed with Fay, we assume the server will always return voucher url.

Reply:
Okay. Sounds like we should treat this as an error if we expect it always be present